src/main/java/com/flightstats/hub/metrics/DataDogWhitelist.java

```diff
@@ -0,0 +1,12 @@
+package com.flightstats.hub.metrics;
```

Comment:
This is a placeholder essentially, the next PR will handle how we will really filter the clients and this will almost definitely disappear